packages/fluent-vue/src/vue/component.ts

```diff
@@ -48,7 +49,13 @@ export default defineComponent({
         return null
       }
 
-      return fluent.formatAttrs(props.path, fluentParams.value)
+      const attrs = fluent.formatAttrs(props.path, fluentParams.value)
+
+      const camelizedAttrs: { [key: string]: string } = {}
```

Comment:
It seems like there's a better way of declaring this type. I'm sorry, I'm totally new to TypeScript

Reply:
There is `Record<string, string>`